README.md

```diff
@@ -45,6 +50,52 @@ There's also a [working example](https://github.com/brightcove/videojs-overlay/b
 
 ## Documentation
 
+### API
+#### `player.overlay()`
+This is the main interface and the way to initialize this plugin. It takes [an options object as input](#plugin-options).
+
+#### `overlay.get()`
+
+Returns an array of all the overlays set up for the current video.
+
+#### `overlay.add(Object|Array)`
+
+Adds one or more overlays to the current list of overlays without replacing the current list of overlays.
+Returns a reference to the added overlays.
+
+```js
+var overlay = player.overlay({
```

Comment:
If there isn't a specific reason to use vars in the new examples, we can use const/let.

Reply:
I can change that right now 😁